CHANGELOG.md

```diff
@@ -7,6 +7,24 @@
 
 * Ported PyGBe to Python 3 (!!!).  This breaks Python 2 support, but who cares.
 * Better regression tests (faster, anyway) using pytest
+* Localized Surface Plasmon Resonance application:
+
+    #### Added:
```

Comment:
I wasn't sure how to indicate that those additions and changes are related to the lspr application, but I guess yes we can, put in the same level.